daemon/execdriver/lxc/driver.go

```diff
@@ -524,3 +524,8 @@ func (t *TtyConsole) Close() error {
 func (d *driver) Exec(c *execdriver.Command, processConfig *execdriver.ProcessConfig, pipes *execdriver.Pipes, startCallback execdriver.StartCallback) (int, error) {
 	return -1, ErrExec
 }
+
+func (d *driver) Stats(id string) (*execdriver.ResourceStats, error) {
+	return nil, fmt.Errorf("container stats are not support with LXC")
```

Comment:
I'll have to see if I can recreate the cgroup hierarchy then we will be able to read out the stats with libcontainer.